obspy/io/resp/scripts/overlaybodes.py

```diff
@@ -0,0 +1,88 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+"""
+Overlay bode plots from multiple responses
+rdseed -R RESP files
+As well as camparing to NRL resp files: 1 dl, 1 sensor.
```

Comment:
We should check whether it might make sense to move plotting stuff to `obspy.imaging`.. not sure right now, there might already be stuff for bode plots there..? (or it might be with `Response`, no time to check right now)